index.html

```diff
@@ -379,6 +385,58 @@
   </dl>
 </section>
 
+<section>
+  ## Events
+
+  ```webidl
+    dictionary SerialConnectionEventInit : EventInit {
+      required SerialPort port;
+    };
+
+    [Exposed=(DedicatedWorker,Window), SecureContext]
+    interface SerialConnectionEvent : Event {
+      constructor(DOMString type, SerialConnectionEventInit eventInitDict);
+      [SameObject] readonly attribute SerialPort port;
+    };
+  ```
+
+  <pre class="example js" title="Keep UI in sync with connected and disconnected serial ports">
+    navigator.serial.addEventListener('connect', event => {
+      // Add |event.port| to the UI.
+    });
+
+    navigator.serial.addEventListener('disconnect', event => {
+      // Remove |event.port| from the UI.
+    });
+  </pre>
+
+  ### Serial port connection
+
+  When the UA detects a new serial port connected to the host it MUST
+  perform the following steps for each script execution environment:
+
+  1. Let |port| be the {{SerialPort}} object representing the new serial port
```

Comment:
Specify the types for variables: `|port:SerialPort|` and `|event:SerialConnectionEvent|`

Reply:
Fixed in https://github.com/reillyeon/serial/pull/5/commits/4bd2b1f5c8789fc00898c9c388b8f1f1116c3340